### DISCUSSION TITLE
Hi All I need Help , My Github account got hacked !!

### DISCUSSION THREAD

**Discussion by CryptoStarWar** [community/community#140031] [Other Feature Feedback, Questions, & Ideas] [answered]
on 26 September i got 2 email in morning,
1 -> Hey ----------! 

An email address ([-------@lamesamail.com]) was added to your account. Visit https://github.com/settings/emails to review email addresses currently associated with your account.

After 5  minute i got email like 

2->Hey ------!

An email address ([-------@gmail.com]) was removed from your account. Visit https://github.com/settings/emails to review email addresses currently associated with your account.

But this all activities was not done by me I haven't given access to some one .

Now i am not able to access my github account can u suggest me what should i Do?
(I already raised github support ticket but 2 days gone still no support from them)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @CryptoStarWar, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
